patch/etc/baseline/module-list.txt

```diff
@@ -70,8 +71,9 @@
 /com/squareup/okhttp3/main/okhttp-3.8.1.jar
 /com/squareup/okio/main/okio-1.13.0.jar
 /com/squareup/retrofit2/main/converter-gson-2.2.0.jar
+/com/squareup/retrofit2/main/converter-jackson-2.0.2.jar
 /com/squareup/retrofit2/main/converter-moshi-2.3.0.jar
-/com/squareup/retrofit2/main/retrofit-2.3.0.jar
+/com/squareup/retrofit2/main/retrofit-2.1.0.jar
```

Comment:
Hmm downgrade. I'm guessing the full test suite still hangs together with this change?

Reply:
I moved consul behind pubnub so that the newer retrofit version is kept.